platform.md

```diff
@@ -378,8 +378,8 @@ Usage:
 | `0`       | Success
 | `11`      | Platform API incompatibility error
 | `12`      | Buildpack API incompatibility error
-| `1-10`, `13-99` | Generic lifecycle errors
-| `200-299` | Analysis-specific lifecycle errors
+| `1-10`, `13-19` | Generic lifecycle errors
+| `40-59` | Analysis-specific lifecycle errors
```

Comment:
Maybe we want to go with `40-49` etc to give us some breathing room in case we add more phases (`extender`, `preparer`)?

It's a tradeoff that would allow less room for more specific error codes for the currently-defined phases, but given I think we could live with that constraint given how few error codes the lifecycle implementation currently uses in the defined ranges (I think we always use `200` etc. currently).

Reply:
Updated in https://github.com/buildpacks/spec/pull/196/commits/9cf6c1d9cf21601ce76bdab12ba0c887174bd1ac